### DISCUSSION TITLE
REI is Crash while i finding recipe (i use aternos server)

### DISCUSSION THREAD

**Discussion by H2Playz** [shedaniel/RoughlyEnoughItems#1594] [Q & A]
here crash log
[crash-2567-03-11_12.58.32-client.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/14554297/crash-2567-03-11_12.58.32-client.txt)
this problem only happen on server and i don't know how to fix this

**Comment by CrypticVerse** [comment]
Looks like a problem with alex's caves. Try removing that and see if it crashes still.
